app/src/panels/time-series/index.ts

```diff
@@ -12,28 +12,39 @@ export default definePanel({
 			return;
 		}
 
+		const filter = () => {
+			if (options.range === 'auto') {
+				if (options.filter) {
+					return options.filter;
+				}
+				return {};
+			}
+
+			return {
+				_and: [
+					{
+						[options.dateField]: {
+							_gte: `$NOW(-${options.range || '1 week'})`,
+						},
+					},
+					{
+						[options.dateField]: {
+							_lte: `$NOW`,
+						},
+					},
+					options.filter || {},
+				],
+			};
+		};
+
```

Comment:
Maybe we can change this to something like this:

```js
		const filter: Filter = {
			_and: [options.filter || {}],
		};

		if (options.range !== 'auto') {
			filter._and.push(
				{
					[options.dateField]: {
						_gte: `$NOW(-${options.range || '1 week'})`,
					},
				},
				{
					[options.dateField]: {
						_lte: `$NOW`,
					},
				}
			);
		}
```	

understandably your code would prevent the unnecessary `_and` wrapper when it's auto, but this suggestion is more so to align with the existing similar "push to filter" logics using throughout the app, just for consistency sake. Examples such as:

https://github.com/directus/directus/blob/f77d12eff4b11862415a432ddf3be97a4d4a729e/app/src/composables/use-revisions.ts#L37-L60

https://github.com/directus/directus/blob/f77d12eff4b11862415a432ddf3be97a4d4a729e/app/src/modules/files/routes/collection.vue#L257-L281

Reply:
Sure, it's look cleaner and more consistent. I will update PR with this change.